hw2/02-search.js

```diff
@@ -1,3 +1,64 @@
-// Add your code here
 
-userInput.addEventListener('click', handleClick);
+function handleClick()
+{
+	console.log('Button clicked!');
+	const txt = document.getElementById('userInput');
+	console.log(txt.value);
+	
+	const items = document.querySelectorAll('.card');
+	for (let i = 0; i < items.length; i++) {
+        const card = items[i];
+		var title = card.querySelector('.card-title')
+		if(title.innerHTML.includes(txt.value))
+		{
+			console.log("-"+txt.value+"-");
+			title.innerHTML = title.innerHTML.replace(txt.value,"<mark>"+txt.value+"</mark>")
+			console.log(title.textContent);
+			card.classList.remove("d-none");
```

Comment:
The highlight will continue to be there from previous searches. It doesn't reset.